README.md

```diff
@@ -40,6 +40,11 @@ We'll review all requests and accept them at our discretion. If accepted, your p
 
 ## Open source projects using 1Password Teams
 
+### TSKaigi Association
+
+The TSKaigi Association (Japanese: 一般社団法人 TSKaigi Association) is a nonprofit organization to hold an annual Tech Conference about TypeScript in Japan. 
+https://association.tskaigi.org/
```

Comment:
@OldBigBuddha could you move this entry to the bottom of the document? Thank you.